lang/pypy/Portfile

```diff
@@ -54,8 +71,12 @@ array set module_scripts {
     smtpd.py smtpd
 }
 
-subport pypy36 {
-    set python.branch 3.6
+worksrcdir          pypy-${github.tag_prefix}${version}
+
```

Comment:
When fetching from a generated GitHub archive, the portgroup's default handling should be correct and should not require you to override this:

```suggestion
```